README.md

```diff
@@ -80,10 +82,8 @@ end
  * `aws_iam_group`
  * `aws_iam_policy`
  * `aws_iam_role`
- ...
-
 
-## Tests
+## Development and Testing the AWS Resources Pack
```

Comment:
Developing and Testing the AWS Resources Pack

Reply:
done